Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.7
+FROM python:3.7-alpine
```

Comment:
If you are pulling from python then the version of alpine is no longer fixed, which is the only reason this fork exists.  You can just pull python:3.7-alpine directly into wherever you are using this image